newmiddleware/auth.js

```diff
@@ -0,0 +1,351 @@
+const autoBind = require("auto-bind")
+const express = require("express")
+
+class AuthMiddleware {
+  constructor({ authMiddlewareService }) {
+    this.authMiddlewareService = authMiddlewareService
+    // We need to bind all methods because we don't invoke them from the class directly
+    autoBind(this)
+  }
+
+  noVerify(req, res, next) {
+    next("router")
+  }
+
+  // Create new page in collection
```

Comment:
remove irrelevant comment

Reply:
Removed in https://github.com/isomerpages/isomercms-backend/pull/328/commits/4e5d20221619aeb00ed410d1f0ee3d739d124e0d